site/assets/scss/_callouts.scss

```diff
@@ -27,7 +28,8 @@
 // Variations
 @each $variant in $bd-callout-variants {
   .bd-callout-#{$variant} {
-    --bd-callout-bg: var(--bs-gray-200); // Boosted mod
-    --bd-callout-border: var(--bs-#{$variant}); // Boosted mod
+    --bd-callout-color: var(--bs-emphasis-color); // Boosted mod
+    --bd-callout-bg: var(--bs-secondary-bg); // Boosted mod
```

Comment:
The color might have changed but I'm not sure this is an issue: 
![image](https://user-images.githubusercontent.com/91960143/211282455-ead662da-f8fa-4345-8b16-93f8694775ee.png)
![image](https://user-images.githubusercontent.com/91960143/211282498-01641dd8-dfe5-4192-9928-2422e7b66c2b.png)

Reply:
My solution won't work in dark mode but that's not important for now. Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/fd58df1fef6f48b76aa3809359ec635584e47164 to get the right color back.